_articles/getting-started.md

```diff
@@ -56,12 +54,12 @@ Once you get to the point of having a rough layout blocked out for your map, it'
 
 BMD has made some rather nice beginner Hammer tutorial videos:
 
-* [part 1: Tile Editor](https://www.youtube.com/watch?v=GMvmdnNM6Sc)
-* [part 2: Mesh Basics](https://www.youtube.com/watch?v=grLUv2hUDRY)
-* [part 3: entity basics](https://www.youtube.com/watch?v=ln3ep-k__dk)
+* [Part 1: Tile Editor](https://www.youtube.com/watch?v=GMvmdnNM6Sc)
+* [Part 2: Mesh Basics](https://www.youtube.com/watch?v=grLUv2hUDRY)
+* [Part 3: entity basics](https://www.youtube.com/watch?v=ln3ep-k__dk)
 
 ## Step #3: Scripting and beyond...
 
-Scripting is the next most important part of your addon. It is divided into Lua scripting, and KeyValue scripting. I'm going to go ahead and redirect you to [Noya's Beginner Scripting Guide](/articles/beginners-guide-to-dota-scripting), since it has essentially the same information that would go in this section. 
+Scripting is the next most important part of your addon. It is divided into Lua scripting, and KeyValue scripting. I'm going to go ahead and redirect you to [Noya's Beginner Scripting Guide](https://github.com/ModDota/moddota.github.io/blob/source/_articles/scripting-introduction.md), since it has essentially the same information that would go in this section. 
```

Comment:
```suggestion
Scripting is the next most important part of your addon. It is divided into Lua scripting, and KeyValue scripting. I'm going to go ahead and redirect you to [Noya's Beginner Scripting Guide](scripting-introduction.md), since it has essentially the same information that would go in this section. 
```